### PR TITLE
Added check for duplicated nuetron agents

### PR DESCRIPTION
**Pull request by jlcoello**
In response to issue : https://github.com/blueboxgroup/ursula-monitoring/issues/44

### CODE REVIEW

File: sensu/plugins/check-neutron-agents-duplicate.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    else:
+        p = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True)
+        stdout, stderr = p.communicate()
+
+    stdout = stdout.strip()
+    if p.returncode == 0 and stdout:
+        return stdout
+    else:
+        print("WARN: \"%s\" returned %s: %s" % (command, p.returncode, stderr))
+        sys.exit(STATE_WARNING)
```
Comment (na-gupta) [inline sensu/plugins/check-neutron-agents-duplicate.py]: It doesn't look like the run() function is being used anywhere and can be taken out.

File: sensu/plugins/check-neutron-agents-duplicate.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+def request_token(url, method='GET', retries=2, **kwargs):
```
Comment (na-gupta) [inline sensu/plugins/check-neutron-agents-duplicate.py]: It looks like request_token() and request() are almost identical except a minor difference in the last return statement and we could probably do with just one of these functions?

File: sensu/plugins/check-neutron-agents-duplicate.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    endpoints = {}
+    for service in access['serviceCatalog']:
+        for endpoint in service['endpoints']:
+            endpoints[service['name']] = endpoint.get('internalURL')
+
+    headers = {'Accept': 'application/json', 'X-Auth-Project-Id': args.tenant,
+               'X-Auth-Token': token}
+
+    endpoint = endpoints['neutron'] + '/v2.0/agents.json'
+    agent_list = request(endpoint, headers=headers )
```
Comment (na-gupta) [inline sensu/plugins/check-neutron-agents-duplicate.py:97]: If the neutron agents API fails for any reason, I think we would get back an empty result here, and check_agents() would pass. So might be good to check the returned agent_list is not empty and warn if it is.

**Comment by omgjlk** [comment]
I think this is good for MVP.

**Pull request merged**